web/templates/layout/site.html.eex

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
```

Comment:
Added this in Trello on this card:
https://trello.com/c/phqVVjAQ/10-set-up-fb-insta-twitter-sharing-content